lib/llvm-opt-transformer.ts

```diff
@@ -60,13 +60,13 @@ function DisplayOptInfo(optInfo: LLVMOptInfo) {
     return displayString;
 }
 
-export function processRawOptRemarks(buffer: string, compileFileName: string = ''): LLVMOptInfo[] {
+export function processRawOptRemarks(buffer: string, compileFileName = ''): LLVMOptInfo[] {
```

Comment:
definitely don't like this type removal change

Reply:
For what it's worth, its possible to disable this with https://biomejs.dev/linter/rules/no-inferrable-types/, but I think this is rather fair to keep (see comment w/ Ofek above)